lib/puppet/type/logical_volume.rb

```diff
@@ -74,6 +76,20 @@ def insync?(is)
     end
   end
 
+  newparam(:thin, :boolean => true, :parent => Puppet::Parameter::Boolean) do
+    desc "Set to true to create a thin provisioned logical volume"
+    defaultto false 
+  end
+
+  newparam(:poolmetadatasize) do
+    desc "Change the size of logical volume pool metadata"
```

Comment:
Summary from IRC discussion:  This is better documented and changed to a property in the future.